app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt

```diff
@@ -394,7 +394,8 @@ private fun GroupConversationDetailsContent(
                         openFullListPressed = openFullListPressed,
                         onAddParticipantsPressed = onAddParticipantsPressed,
                         onProfilePressed = onProfilePressed,
-                        lazyListState = lazyListStates[pageIndex]
+                        lazyListState = lazyListStates[pageIndex],
+                        allowAddMember = conversationSheetState.conversationSheetContent?.title?.isNotEmpty() ?: false
```

Comment:
I would change this to check if the name is not null and there is only one member, since legacy group conversations did allow empty names
this way, legacy groups will still continue to work as before.